README.md

```diff
@@ -30,7 +30,7 @@ npm install
 npm start
 ```
 
-This will start the development server on `http://localhost:8080/en/`. This should reload automatically when you make changes to the code, but no code is perfect, so sometimes you may need to restart it. :)
+This will start the development server on `http://localhost:8080/en/`. This link should reload automatically when you make changes to the code, but no code is perfect, so sometimes you may need to restart it. :)
```

Comment:
```suggestion
This will start the development server on `http://localhost:8080/en/`. This page should reload automatically when you make changes to the code, but no code is perfect, so sometimes you may need to restart it. :)
```